src/test/java/ScrabbleGameTest.java

```diff
@@ -22,7 +22,14 @@ public void playWord_true()
         ScrabbleGame game = new ScrabbleGame(dictionary, letterPool);
 
         // when
-        boolean val = game.playWord(word);
+        boolean val;
+        try
+        {
+            val = game.playWord(word);
+        } catch (Exception e)
+        {
+            val = false;
+        }
```

Comment:
You don't need the try/catch.
Just declare that the test method throws the Exceptions.

Reply:
When I do this, my tests say that they fail. How can I catch an exception being thrown by a test?

Reply:
This test method shouldn't be throwing an exception, right?

Reply:
public void methodName() throws SpecificException